### PR TITLE
chore(Elixir): Fix compiler warnings

### PR DESCRIPTION
**Pull request by tomciopp**
@lau Let me know if you have any questions.

### CODE REVIEW

**Comment by bcardarella** [comment]
Is there anything holding up the merging of this PR? Would be nice to silence the deprecation warnings

File: lib/tzdata/ets_holder.ex [hunk 1]
```diff
@@ -8,7 +8,7 @@ defmodule Tzdata.EtsHolder do
 
   @file_version 2
 
-  def start_link() do
+  def start_link(_) do
```
Comment (lau) [inline lib/tzdata/ets_holder.ex:11]: Thoughts on accepting any argument and ignoring it vs only allowing a certain argument? Or using the argument and passing it on to the GenServer.start_link function.

**Comment by lau** [comment]
> Is there anything holding up the merging of this PR? Would be nice to silence the deprecation warnings

It has just been some other things going on. But I should have time to fix it and do a release today or within a few days.

**Comment by lau** [comment]
Closing because https://github.com/lau/tzdata/pull/106 has been merged with similar fixes. Version 1.1.0 has been released.

**Pull request closed without merging**